vowpalwabbit/cbify.cc

```diff
@@ -145,8 +139,7 @@ void copy_example_to_adf(cbify& data, example& ec)
     // offset indices for given action
     for (features& fs : eca)
     {
-      for (feature_index& idx : fs.indicies)
-      { idx = ((((idx >> ss) * 28904713) + 4832917 * (uint64_t)a) << ss) & mask; }
+      for (feature_index& idx : fs.indicies) { idx = (((idx >> ss) * adf_data.num_actions + a) << ss) & mask; }
```

Comment:
@pmineiro wouldnt this change regress models using the previous way since the weight is located elsewhere now (different hash)?

Reply:
Yes and no.  

**No**: `--cbify` is  used for testing implementations against a supervised dataset, and the details of how the features were induced were private already, so nobody was saving and reusing the model.  Therefore ignore this issue.

**Yes**: Your comment does apply to the (previous implementation of) `--cb`, which is why the model load of older model versions needs to be modified.